packages/node_modules/@ciscospark/react-component-activity-item-base/src/index.js

```diff
@@ -74,17 +77,20 @@ function ActivityItemBase({
     onActivityRetry(id);
   }
 
-  function getActionClassNames(highlight) {
+  function getActionClassNames(highlight, actionPending) {
     const actionClassNames = ['activity-post-action', styles.activityPostAction];
     if (highlight) {
       actionClassNames.push('isHighlighted', styles.isHighlighted);
     }
+    if (actionPending) {
+      actionClassNames.push('pending');
```

Comment:
Can we be more specific here since the `actionPending` is really only used or `flagPending`?

Reply:
Yea that could work, especially if we switch to obj destructuring